app/src/main/java/com/kks/nimblesurveyjetpackcompose/ui/presentation/survey/SurveyEmojiQuestion.kt

```diff
@@ -1,31 +1,42 @@
 package com.kks.nimblesurveyjetpackcompose.ui.presentation.survey
 
 import androidx.compose.foundation.layout.Arrangement
-import androidx.compose.foundation.layout.Box
 import androidx.compose.foundation.layout.Row
 import androidx.compose.foundation.layout.fillMaxWidth
-import androidx.compose.foundation.layout.size
-import androidx.compose.material.ButtonDefaults
 import androidx.compose.material.Text
 import androidx.compose.material.TextButton
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
 import androidx.compose.runtime.mutableStateOf
 import androidx.compose.runtime.remember
-import androidx.compose.ui.tooling.preview.Preview
-import androidx.compose.runtime.*
+import androidx.compose.runtime.setValue
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.sp
+import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType
 import com.kks.nimblesurveyjetpackcompose.model.SurveyAnswer
 import com.kks.nimblesurveyjetpackcompose.ui.theme.Black50
 
 private val SMILEY_EMOJIS = listOf("😡", "😕", "😐", "🙂", "😄")
+private val THUMBS_EMOJIS = listOf("\uD83D\uDC4D", "\uD83D\uDC4D", "\uD83D\uDC4D", "\uD83D\uDC4D", "\uD83D\uDC4D")
 
 @Composable
-fun SurveySmileyQuestion(answers: List<SurveyAnswer>, onChooseAnswer: (answers: List<SurveyAnswer>) -> Unit) {
-    var selectedIndex by remember { mutableStateOf(-1) }
+fun SurveyEmojiQuestion(
+    answers: List<SurveyAnswer>,
+    questionDisplayType: QuestionDisplayType,
+    onChooseAnswer: (answers: List<SurveyAnswer>) -> Unit
+) {
+    var selectedIndex by remember { mutableStateOf(answers.indexOfFirst { it.selected }) }
+    val isShowAllLeftEmoji = questionDisplayType != QuestionDisplayType.SMILEY
+
     Row(modifier = Modifier.fillMaxWidth(), horizontalArrangement = Arrangement.Center) {
-        SMILEY_EMOJIS.forEachIndexed { index, text ->
+
+        when (questionDisplayType) {
+            QuestionDisplayType.SMILEY -> SMILEY_EMOJIS
+            QuestionDisplayType.THUMBS -> THUMBS_EMOJIS
+            else -> listOf()
```

Comment:
Could use `emptyList()` and imports here 
```suggestion
            SMILEY -> SMILEY_EMOJIS
            THUMBS -> THUMBS_EMOJIS
            else -> emptyList()
```

Reply:
"and imports" 😉 
```kotlin
            SMILEY -> SMILEY_EMOJIS
            THUMBS -> THUMBS_EMOJIS
```

Reply:
@ryan-conway I already tried doing that but I am using `QuestionDisplayType` at
```
fun SurveyEmojiQuestion(
    answers: List<SurveyAnswer>,
    questionDisplayType: QuestionDisplayType,
    onChooseAnswer: (answers: List<SurveyAnswer>) -> Unit
) {
```
So the imports will be like
```
import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType
import com.kks.nimblesurveyjetpackcompose.model.QuestionDisplayType.*
```
In my opinion, I am importing the same thing twice. Should I import like that?

Reply:
@AVI5HEK thoughts? 🤔

Reply:
@kaungkhantsoe not sure if I understood correctly. We're not importing the same things twice. 😄 First, we're importing `QuestionDisplayType`, and then we're importing its children.